README.md

```diff
@@ -314,10 +314,10 @@ To add a class to the input group wrapper, use the `:input_group_class` option.
 
 Bootstrap mark-up dictates that most input field types have the label and input wrapped in a `div.mb-3`.
 
-If you want to add an additional CSS class or any other attribute to the form group div, you can use the `wrapper: { class: 'additional-class', data: { foo: 'bar' } }` option.
+If you want to change the CSS class or any other attribute to the form group div, you can use the `wrapper: { class: 'mb-3 additional-class', data: { foo: 'bar' } }` option.
 
 ```erb
-<%= f.text_field :name, wrapper: { class: 'has-warning', data: { foo: 'bar' } } %>
+<%= f.text_field :name, wrapper: { class: 'mb-3 has-warning', data: { foo: 'bar' } } %>
```

Comment:
Yes, that is correct.  It is unfortunate, but I see no way around it.  Do you?

Reply:
One idea I had was to start providing some way to change the "default" behaviour of `bootstrap_form` in some ways. In this case, we'd provide a configuration option that would allow us to set the for wrapper class(es) for all forms.

We'd then probably want to provide the same change of behaviour at the form level -- with some way of changing the default configuration for the duration of the form.

If they need to change it for a single field in a form, then your solution is fine.

RIght now I don't have time to expand on the thought -- sorry. But hopefully that gives you something to think about.

If we do something like I described above, I would suggest we do a sequence of PRs. I think what I'm proposing is too big for one PR.

Reply:
👍